### ISSUE TITLE
error chemical spieces in ver 0.10.0

### ISSUE THREAD

**Issue by thangckt** [MDIL-SNU/SevenNet#104]
hi @YutackPark 

with new update in ver 0.10.0m the below setting
```
chemical_species: ['Mo', 'W']
```
will cause error:
```
 File "path/lib/python3.11/site-packages/sevenn/parse_input.py", line 221, in read_config_yaml
    model_meta = init_model_config(config)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "path/lib/python3.11/site-packages/sevenn/parse_input.py", line 67, in init_model_config
    elif 'univ' in input_chem.lower():
                   ^^^^^^^^^^^^^^^^
AttributeError: 'list' object has no attribute 'lower'
```
can you have a look?
Thanks.

**Comment by YutackPark** [comment]
Shame that this kind of error remains, fixed with #105

**Comment by thangckt** [comment]
@YutackPark, It works now :)
Thanks
